environments/centos-8/Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN set -x && \
     dnf install -y --enablerepo epel --enablerepo powertools \
         findutils gcc gcc-c++ make patch bzip2 curl autoconf automake \
         openssl-devel libyaml-devel libffi-devel readline-devel zlib-devel \
-        gdbm-devel ncurses-devel && \
+        gdbm-devel ncurses-devel \
+        rust-toolchain && \
```

Comment:
Sorry, I was wrong. It's apparently rust-toolset, not rust-toolchain.

Reply:
Fixed at f602878

Reply:
Fixed at f602878